lib/index.js

```diff
@@ -1,7 +1,8 @@
 let buildBundler = require("./build-bundler");
 
-module.exports = (pluginConfig, assetManager, { watcher, browsers, compact }) => {
-	let config = buildConfig(pluginConfig, assetManager, { browsers, compact });
+module.exports = (pluginConfig, assetManager, options = {}) => {
+	let { watcher } = options;
+	let config = buildConfig(pluginConfig, assetManager, options);
```

Comment:
This would also change the behaviour by passing in `watcher` as an additional option if `buildConfig` cared.  Is that intentional?